action.yml

```diff
@@ -49,7 +49,7 @@ runs:
     - name: Set Action Version
       shell: bash
       run: |
-        echo "version=0.0.1" >> $GITHUB_ENV
+        echo "version=0.0.2" >> $GITHUB_ENV
```

Comment:
Does this need a new version given it's just documentation fixes?

Reply:
Ah I guess if a new version isn't published, the Marketplace listing will show the outdated information?

Reply:
Hmm. I'll give it go without bumping the version.

Reply:
Maybe. I deleted the existing tag, pushed a new one pointing at @HEAD, then deleted + recreated the release. Seems fine?